test/invites.service.spec.ts

```
@@ -0,0 +1,197 @@
+import { NotificationsService } from '../src/notifications/notifications.service';
+import { ConfigService } from '@nestjs/config';
+import { CacheService } from '../src/redis/cache.service';
+import { TensorflowPredictionService } from '../src/business/services/tensorflow-prediction.service';
+import { Role } from '@/auth/enums/role.enum';
+import { BusinessUserRole } from '@/business/enums/business-user-role.enum';
+import { Types } from 'mongoose';
+import { NotFoundException, BadRequestException } from '@nestjs/common';
+
+describe('BusinessService (Invites)', () => {
+  let service: BusinessService;
+  let mockBusinessInviteModel: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Move arrow function 'createMockQuery' to the outer scope.

(unicorn/consistent-function-scoping)